Dockerfile

```diff
@@ -20,6 +20,9 @@ FROM nginx:1.25.3-alpine
 
 ENV KAFKA_CONNECT_CLUSTERS=http://localhost:8083/
 
+RUN rm -rf /etc/nginx/conf.d/*.conf
+COPY ./nginx.config/default.conf /etc/nginx/conf.d/defualt.conf
```

Comment:
@yahyaAlsaidi could you rename the `nginx.config/` dir to `etc/nginx/` or at least to `etc/`?

Reply:
Thanks will do, but what the  reason behind make the `nginx.config/` dir renamed to `/etc/nginx/`?

Reply:
@yahyaAlsaidi it's only a convention that more and more projects use to store config files, similar to how Linux stores config. That way it's also easy to understand where those files belong.

Reply:
Ah, got it! Now it makes sense. I knew it was a convention, but I was waiting for your reply to get a clearer understanding of it. Thanks for clarifying!

Reply:
You are welcome and thx a lot for your effort! :pray: